test/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Distributed under the Boost Software License, Version 1.0.
 # See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt
 
-find_package(PythonInterp)
+find_package(Python3)
 if (PYTHONINTERP_FOUND)
   # checks that b2 and cmake are in sync
   add_test(NAME runpy-${PROJECT_NAME}_check_build_system COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/check_build_system.py)
```

Comment:
You should be able to use the target here, `Python3::Interpreter`. If not, `${Python3_EXECUTABLE}` is the new variable name.

Reply:
`find_package(Python3::Interpreter)` did not work, but `find_package(Python3 COMPONENTS Interpreter)` does. I will implement that in another minor PR.

Reply:
That’s the default so it’s fine either way, what I meant was `${Python3_EXECUTABLE}` (the variable) can be replaced by `Python3::Interpreter` (a CMake target).

Reply:
(After COMMAND a target is supported)

Reply:
Ok, I see.